packages/core/src/types/externalMatch.ts

```diff
@@ -26,6 +26,7 @@ export type ExternalSettlementTx = {
   data: `0x${string}`
   accessList: AccessList
   gas?: `0x${string}`
+  value?: `0x${string}`
```

Comment:
To confirm, this field is for native ETH external matches?

Reply:
yes, for native ETH sells